src/appleseed.shaders/src/maya/as_maya_anisotropic.osl

```diff
@@ -269,7 +269,7 @@ shader as_maya_anisotropic
                 );
         }
 
-        if (transparency)
+        if (max(transparency) > 0.0)
```

Comment:
@est77 i agree with if (color), we talked about this last week. i just had no time yet to replace all those tests, but to recap, i'll leave if (foo) only and only for boolean types, which in OSL means ints that can only take 0 or 1 really since we have no bool.
We cannot test if (color > 0), OSL fails with this expression, but we can test if (color != 0), that was my suggestion.